src/pip/_internal/download.py

```diff
@@ -889,6 +890,19 @@ def unpack_url(
         write_delete_marker_file(location)
 
 
+def parse_content_disposition(content_disposition, default_filename):
+    # type: (str, str) -> str
+    """
+    Parse filename from content_disposition.
+    return default_filename if parsed result is empty
+    """
+    type, params = cgi.parse_header(content_disposition)
```

Comment:
It would be better to choose a variable name other than `type` since `type` is a built-in function.